tests/foreman/longrun/test_inc_updates.py

```diff
@@ -147,17 +141,18 @@ def host(
     module_target_sat,
 ):
     # Create client machine and register it to satellite with rhel_7_partial_ak
-    rhel7_contenthost_module.install_katello_ca(module_target_sat)
+    # rhel7_contenthost_module.install_katello_ca(module_target_sat)
     # Register, enable tools repo and install katello-host-tools.
-    rhel7_contenthost_module.register_contenthost(
-        module_entitlement_manifest_org.label, module_ak.name
+    rhel7_contenthost_module.register(
+        module_sca_manifest_org, None, module_ak.name, module_target_sat
     )
-    rhel7_contenthost_module.enable_repo(REPOS['rhst7']['id'])
+    rhel7_contenthost_module.enable_repo(REPOS['rhsclient7']['id'])
     rhel7_contenthost_module.install_katello_host_tools()
     # make a note of time for later wait_for_tasks, and include 4 mins margin of safety.
     timestamp = (datetime.utcnow() - timedelta(minutes=4)).strftime('%Y-%m-%d %H:%M')
     # AK added custom repo for errata package, just install it.
     rhel7_contenthost_module.execute(f'yum install -y {FAKE_4_CUSTOM_PACKAGE}')
+    rhel7_contenthost_module.execute('subscription-manager repos')
```

Comment:
What's the purpose of this step?

Reply:
To get the package profile uploaded and errata recalculated on the Satellite side.

Reply:
AFAIK `katello-package-upload` is for compatibitlity with older version of `subscription-manager`
In this test `katello-package-upload` didn't return anything after execution so I used `subscription-manager repos`